DatabaseExplorer/MySqlDbAdapter.h

```diff
@@ -83,7 +83,7 @@ class MySqlDbAdapter : public IDbAdapter {
 	wxString m_userName;
 	wxString m_password;
 
-	DatabaseLayer* m_pDbLayer;
+	DatabaseLayerPtr m_pDbLayer;
```

Comment:
Possibly `std::weak_ptr` to mimic more old behavior to not increase ref-counting.